src/Tizen.Wearable.CircularUI.Forms.Renderer/RotaryService.cs

```diff
@@ -0,0 +1,53 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using ElmSharp.Wearable;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(Tizen.Wearable.CircularUI.Forms.Renderer.RotaryService))]
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class RotaryService : IRotaryService
+    {
+        event EventHandler<RotaryEventArgs> _rotated;

```

Comment:
It looks good to use Delegate instead of Event.
There is no consequential difference, but it looks more intuitive.

Reply:
Ok, sounds fair. Updated.